update-codeowners-from-packages/action.yaml

```diff
@@ -80,6 +74,13 @@ runs:
 
             echo "$line" >>.github/CODEOWNERS
         done
+
+        # Add manually defined CODEOWNERS
+        echo "### Copied from ${{ inputs.codeowners-manual }} ###" >>.github/CODEOWNERS
+        touch ${{ inputs.codeowners-manual }}
+        cat ${{ inputs.codeowners-manual }} >> .github/CODEOWNERS
+        echo "" >>.github/CODEOWNERS
```

Comment:
```suggestion
        echo "" >>.github/CODEOWNERS
        echo "### Copied from ${{ inputs.codeowners-manual }} ###" >>.github/CODEOWNERS
        cat ${{ inputs.codeowners-manual }} >> .github/CODEOWNERS
```

Reply:
Thanks! 
I have updated the location of echo. 
Should we remove `touch` command as well?

I was guessing it was there to avoid failure if the user didn't make the file for manual codeowners.

Reply:
Oh, sorry. I misunderstood it as initializing the output file.